src/modules/esl-share/actions/copy-action.ts

```diff
@@ -0,0 +1,39 @@
+import {memoize} from '../../esl-utils/decorators';
+import {ESLShareBaseAction} from '../core/esl-share-action';
+import {ESLEventUtils} from '../../esl-utils/dom/events';
+
+import type {ESLShareButton} from '../core/esl-share-button';
+import type {AlertActionParams} from '../../esl-alert/core';
+
+@ESLShareBaseAction.register
+export class ESLShareCopyAction extends ESLShareBaseAction {
+  public static readonly is: string = 'copy';
+
+  @memoize()
+  public get isAvailable(): boolean {
+    return navigator.clipboard !== undefined;
+  }
+
+  protected get alertText(): string {
+    return 'Copied to clipboard';
```

Comment:
Cosmetic: we need to think about more accessible or prepared value by default.
It is strange if we give an action that the user needs to overwrite anyway (as it is not internalized|prepared|etc.) if we are talking about prod usage.

Reply:
Discussed (02.03)

Alert config moved as additionalParams to the action configuration
Passs this param to the button as an `additionalParams` field (@prop / @jsonParam)